watershed_workflow/io.py

```diff
@@ -103,12 +123,17 @@ def write_dataset_to_hdf5(filename, dataset, attributes, time0=None):
             'Raster cannot be written as transform is not rectilinear, which is an assumption of simulators.'
         )
     x = np.array([(transform * (i, 0))[0] for i in range(profile['width'])])
-    y = np.array([(transform * (0, j))[0] for j in range(profile['height'])])
-
+    y = np.array([(transform * (0, j))[1] for j in range(profile['height'])])
```

Comment:
oh goodness, how has no one complained about this yet?  Thanks, this is kinda an important bug!